app/views/graders/_ta_grade_distribution_graph.html.erb

```diff
@@ -1,46 +1,63 @@
 <% unless ta.nil? %>
-  <h3><%= ta.first_name + ' ' + ta.last_name%></h3>
-  <div>
-    (<%= t('browse_submissions.how_many_marked',
-           num_marked: assignment.get_num_marked(ta.id),
-           num_assigned: assignment.get_num_assigned(ta.id)) %>)
-  </div>
-  <canvas id='a<%= assignment.id %>_ta_<%= ta.id %>' width='400' height='350'></canvas>
-  <script>
-    //<![CDATA[
-    // Set up graph
-    var ctx = document.getElementById('a<%= assignment.id %>_ta_<%= ta.id %>').getContext('2d');
 
-    // Set up data
-    var data = {
-      // Set up labels [0, 5, ..., 95]
-      labels: Array.apply(null, Array(20)).map(function (_, i) { return i * 5; }),
-      datasets: [{
-        data: <%= ta.grade_distribution_array(assignment, 20) %>
-      }]
-    };
+    <% if ta.nil? || ta.first.nil? %>
+        <% @ta_id = 0 %>
```

Comment:
The indentation in this file is off. You should use 2 spaces for indentation (everywhere).